dbt/README.md

```diff
@@ -0,0 +1,79 @@
+- `models/`: Directory containing SQL models for dbt.
+  - `docs.md`: Documentation for the models.
+  - [`overview.md`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fgmendoza%2Fsource%2Fcivis-dbt-docker%2Fplatform-code-examples%2Fdbt%2Fmodels%2Foverview.md%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/gmendoza/source/civis-dbt-docker/platform-code-examples/dbt/models/overview.md"): Overview of the dbt project.
+  - `schema.yml`: Schema configuration for the models.
+  - `staging/`: Directory containing staging models.
+    - `stg_customers.sql`: SQL model for staging customers.
+    - `stg_orders.sql`: SQL model for staging orders.
+    - `stg_payments.sql`: SQL model for staging payments.
+- `seeds/`: Directory containing seed files for dbt.
+
+## Configuration
+
+### dbt_project.yml
+
+The [`dbt_project.yml`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fgmendoza%2Fsource%2Fcivis-dbt-docker%2Fplatform-code-examples%2Fdbt%2Fdbt_project.yml%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/gmendoza/source/civis-dbt-docker/platform-code-examples/dbt/dbt_project.yml") file contains the main configuration for the dbt project, including the project name, version, paths to models, seeds, and other configurations.
+
+### profiles.yml
+
+The [`profiles.yml`](command:_github.copilot.openRelativePath?%5B%7B%22scheme%22%3A%22file%22%2C%22authority%22%3A%22%22%2C%22path%22%3A%22%2FUsers%2Fgmendoza%2Fsource%2Fcivis-dbt-docker%2Fplatform-code-examples%2Fdbt%2Fprofiles.yml%22%2C%22query%22%3A%22%22%2C%22fragment%22%3A%22%22%7D%5D "/Users/gmendoza/source/civis-dbt-docker/platform-code-examples/dbt/profiles.yml") file contains the profiles configuration for dbt, specifying different environments (e.g., development, production) and their settings, such as database credentials and connection details.
+
+
+## Platform Configuration
+
+When using this example in Civis Platform, you will need to include `dbt` for both `dbt Project Directory` and `dbt Profiles Directory` as highlighted in the screenshot below.
+
+![Civis DBT Screenshot](./civis_dbt_screenshot.png)
+
+When creating a dbt job with your own git repository, if `profiles.yml` and/or `dbt_project.yml` are located at the root of the repository, you can leave these blank.
+
+## Environment Variables
+
+Below is a quick reference of the environment variables used in the dbt example. For in depth documentation on the topic, please see the [dbt's environment variables documentation.](https://docs.getdbt.com/docs/build/environment-variables)
```

Comment:
are any of these environment variables listed below the dbt-used ones? or should this be an additional note?

```suggestion
Below is a quick reference of the fundamental environment variables used in the dbt example code, which are provided by Platform when run in that context. For in depth documentation on the topic, please see [our documentation](https://support.civisanalytics.com/hc/en-us/articles/27926077597581-dbt-Scripts). 

 For more information on environment variables automatically used by dbt (which can also be specified from Platform), see [dbt's environment variables documentation.](https://docs.getdbt.com/docs/build/environment-variables)
```